apps/sms/services/activity/client.js

```diff
@@ -0,0 +1,62 @@
+/* global EventDispatcher,
+          threads
+*/
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for managing activities.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'activity-service';
+
+  /**
+   * Definition of all available private members.
+   * @type {Object}
+   */
+  const priv = Object.freeze({
```

Comment:
Yeah, maybe it's just me, but I feel like Symbol suits "private case" better. Some time ago I wanted to switch to it after reading:

"Symbols are a new, special kind of object that can be used as a unique property name in objects.
Using symbols instead of strings allows different modules to create properties that don't conflict with one another.
Symbols can also be made private, so that their properties can't be accessed by anyone who doesn't already have direct access to the symbol." [1]

But something changed since then and symbols aren't really private and can be accessed via "getOwnPropertySymbols", but still looks ok for our purpose + we can add private prototype method simpler and add jsdoc for every private member if we want.

I use it in my personal projects and eager to try in sms, but only if you guys agree :)

[1] http://tc39wiki.calculist.org/es6/symbols/